### PR TITLE
refactor: DLT-2122 move css out of recipes

### PR DESCRIPTION
**Pull request by juliodialpad**
# Move CSS within vue recipes to dialtone-css

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [x] Refactor

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-2122

## :book: Description

- Moved all CSS within recipes to dialtone-css
- Renamed classes that were not using BEM (Maybe breaking change)

## :bulb: Context

We're moving all CSS defined in dialtone vue to dialtone-css.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.

For all Vue changes:

- [x] I have added / updated unit tests.
- [x] I have made my changes in Vue 2 and Vue 3. Note: you may sync your changes from Vue 2 to Vue 3 (or vice versa) using the `./scripts/dialtone-vue-sync.sh` script. Read docs here: [Dialtone Vue Sync Script](../packages/dialtone-vue3/.github/CONTRIBUTING.md#dialtone-vue-sync-script)

For all CSS changes:

- [x] I have used design tokens whenever possible.

## :crystal_ball: Next Steps

Visually test and review all the recipes to check that everything's still working as expected.

### CODE REVIEW

**Comment by iropolo** [comment]
I will take a look to this today.

File: packages/dialtone-vue2/recipes/cards/ivr_node/ivr_node.mdx [hunk 1]
```diff
@@ -250,7 +250,7 @@ A node in IVR like branch that has a condition instead of dtmf connector.
   :node-type="branch"
 >
   <template #connector>
-    <div class="ivr-connector d-px8 d-h24 d-bar-pill d-mbn12 d-fc-neutral-white d-fs-100">Add branch</div>
+    <div class="dt-recipe-ivr-node__connector d-px8 d-h24 d-bar-pill d-mbn12 d-fc-neutral-white d-fs-100">Add branch</div>
```
Comment (francisrupert) [inline packages/dialtone-vue2/recipes/cards/ivr_node/ivr_node.mdx:253]: Should these CSS utilities values be folded into `.dt-recipe-ivr-node__connector`? There are a few others like that throughout this diff.

**Review by francisrupert — COMMENTED** [review]
Wow, that's a lotta work, but quite necessary. I didn't go through each line, but here are a few themes I see repeated throughout:

**1.** There's quite a bit of nesting going on, resulting in over-specific CSS selectors. Unless there's a really good reason, keep them as flat as possible, e.g. 
```css
.dt-recipe-attachment-carousel__progress-bar {
  ...
  .dt-recipe-attachment-carousel__progress-bar__circle {
    ...
  }
}
```    
...can probably be...
```css
.dt-recipe-attachment-carousel__progress-bar {
  ...
}
.dt-recipe-attachment-carousel__progress-bar__circle {
  ...
}
```
...or perhaps...
```css
.dt-recipe-attachment-carousel {
  &__progress-bar {
    ...
  }
  &__progress-bar__circle {
    ...
  }
}
```
**2.** The above example also has some double-BEM'ing going on wit `__`. To take the above as an example, perhaps...
```css
.dt-recipe-attachment-carousel {
  &__progress-bar {
    ...
  }
  &__progress-bar-circle {
    ...
  }
}
```
It's a case by case judgement call, and naming... things is fun... and hard. 

**3.** Should `data-qa` values also match BEM convention? I don't have an opinion on that. e.g. 
```
class="dt-recipe-feed-item-row__left-time"
data-qa="dt-recipe-feed-item-row--left-time"
```

**Comment by iropolo** [comment]
Should be ok, the remaining percy error is an icon update (expected).

edit: didn't see Fracis comment. My comment was regarding fixing PR percy errors.

**Comment by braddialpad** [comment]
> Should be ok, the remaining percy error is an icon update (expected).

Seems like there's an additional border problem on the latest build

File: packages/dialtone-vue2/recipes/cards/ivr_node/ivr_node.mdx (hunk 1 above)
Comment (braddialpad) [reply packages/dialtone-vue2/recipes/cards/ivr_node/ivr_node.mdx:253]: yeah I think so for sure

**Review by braddialpad — APPROVED** [review]
Since this is such a big change with high potential for conflicts, I'd rather get this into the main branch and make francis' requested changes in a separate PR. The only thing we want to change before merging here are any further visual regressions.

**Comment by braddialpad** [comment]
Merged in staging, going to see if it fixes the final percy error

**Comment by braddialpad** [comment]
Alright looks good! I'm going to merge this and create a separate "clean up" PR

**Pull request merged**